tests/03_db/test_db.py

```diff
@@ -63,3 +64,59 @@ async def test_sync_db(db_sync, db):
 )
 def test_DB_ENGINE_is_postgres():
     pass
+
+
+async def test_DB_class_async():
+
+    try:
+        DB._engine_async
+        delattr(DB, "_engine_async")
+    except AttributeError:
+        pass
+    with pytest.raises(AttributeError):
+        DB._engine_async
```

Comment:
I'd say that an `assert` in front of this kind of statements make them a bit more readable (also below)